tl.lua

```diff
@@ -4296,6 +4297,74 @@ function tl.pretty_print_ast(ast, gen_target, mode)
             return out
          end,
       },
+      ["string"] = {
+         after = function(node, children)
+
+
+
+
+
+
+            if node.tk:sub(1, 1) == "[" or gen_target ~= "5.1" then
```

Comment:
```suggestion
            if node.tk:sub(1, 1) == "[" or gen_target ~= "5.1" or not string.find(node.tk, "\\", 1, true) then
```

I would still add this bit just to stay on the safe side performance-wise